app/models/manageiq/providers/ansible_tower/configuration_manager/ansible_tower_job.rb

```diff
@@ -0,0 +1,60 @@
+class ManageIQ::Providers::AnsibleTower::ConfigurationManager::AnsibleTowerJob < ::OrchestrationStack
+  require_dependency 'manageiq/providers/ansible_tower/configuration_manager/ansible_tower_job/status'
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::ConfigurationManager"
+  belongs_to :job_template, :foreign_key => :orchestration_template_id, :class_name => "ConfigurationScript"
+
+  #
+  # Allowed options are
+  #   :limit      => String
```

Comment:
@mkanoor I have not tested the limit. The format should meet whatever the API expects. @syncrou  Can you answer @mkanoor's question?

Please note here in the comment symbols are used, i.e., `:limit`, and `:extra_vars`. This is because all options hash keys are symbols for other types of orchestration stacks. But in ansible_tower_client it expects keys to be string, i.e. `"limit"`. We can enhance the client to accept symbols or force the caller here in this class to pass in a string. I prefer to enhance the client code.